README.md

```diff
@@ -1,26 +1,33 @@
 # OTel Me If It's Right
 
-Checker if the implementation of OpenTelemetry instrumentation is correct.
+Checker for if the implementation of OpenTelemetry instrumentation is correct by scanning the code in your repository.
```

Comment:
```suggestion
This is a *checker* that analyzes if your implementation of OpenTelemetry instrumentation is correct.
```

or something on those lines, without the "by scanning the code in your repository"

Reply:
OK cool - I'll address Gregor's comment, then we can expand it later.